### ISSUE TITLE
When deleting a note, ask what to do with attachments

### ISSUE THREAD

**Issue by amelandri** [fabiospampinato/notable#280]
When deleting a note it would be useful if the app would ask what if the user wants to keep them or if he want to delete them.

**Comment by amelandri** [comment]
I imported all my Evernote notes, and after that I decided to stat from scratch and deleted all notes, but te app did not delete the attachments, so I thought that it was just how it works.

**Comment by fabiospampinato** [comment]
It sounds like a bug, how did you delete the notes? Manually from the file explorer while the app was closed?

**Comment by amelandri** [comment]
No, I selected all of them using the tag automatically created during the import.

I'm not sure if I moved them to trash or deleted permantly but probably the latter.

P.s.
Che app spettacolare! complimenti davvero! :-)

**Comment by RomanKornev** [comment]
Is there any way of **not** removing the attachment when unlinking it right now?
The message box "Are you sure you want to permanently delete ..." should have: 
- "Yes" - deletes attachment from disk
- "No" - unlinks it from metadata but doesn't delete the file
- "Cancel" - same as now

**Comment by fabiospampinato** [comment]
> Is there any way of not removing the attachment when unlinking it right now?

If it's linked to by some other note it's not deleted. I guess the app should ask what to do with the attachment in this situation 🤔
